bin/disco_deploy.py

```diff
@@ -28,6 +29,7 @@
      --hostclass HOSTCLASS  Limit command to a specific hostclass
      --environment ENV      Environment to operate in
      --allow-any-hostclass  Do not limit command to hostclasses defined in pipeline
+     --strategy STRATEGY    The deployment strategy to use. Currently supported: 'classic' or 'blue_green'.
```

Comment:
the docopt website says it does support defaults for arguments http://docopt.org/

`If you want to set a default value for an option with an argument, put it into the option's description, in the form [default: <the-default-value>]`